### ISSUE TITLE
VisionOS Support

### ISSUE THREAD

**Issue by LuisAlvarez12** [AudioKit/AudioKitUI#80]
### Description

At the moment, attempting to use a AudioKitUI View, such as NodeOutputView, results in an issue when building: 
![Screenshot 2024-02-05 at 12 30 07 PM](https://github.com/AudioKit/AudioKitUI/assets/15304780/0a684570-fd92-4af3-9c48-3d9d3ccdc13e)

According the the log, it mentions that views that utilize FloatPlot can not be used for VisionOS since `MTKView` and `MTKViewDelegate` do not support Vision OS:
![Screenshot 2024-02-05 at 12 31 15 PM](https://github.com/AudioKit/AudioKitUI/assets/15304780/3ae5f35b-4b8b-4643-813d-1ab28b0ef1c3)

### Proposed Solution

Ideally an alternative way to use the FloatPane utility without relying on MTKView

### Describe Alternatives You've Considered

N/A

**Comment by aure** [comment]
Yes, we are aware, but thanks for opening the issue, it doesn't hurt.

**Comment by LuisAlvarez12** [comment]
Thank you! Appreciate the hard work @aure
